src/analyses/invariant_set.cpp

```diff
@@ -506,7 +507,7 @@ void invariant_sett::strengthen_rec(const exprt &expr)
 
       forall_operands(it, bitand_op)
       {
-        exprt tmp(equal_expr);
```

Comment:
why `auto`?

Reply:
That means that the type of `tmp` will allow us to access .opX().